CHANGELOG.rst

```diff
@@ -48,7 +47,13 @@ Internal changes:
 - Add a container class for the coverage data. (:issue:`1023`)
 - Replace setup.py with hatchling. (:issue:`1026`)
 - Move gcovr to ``src`` directory. (:issue:`1027`)
-- Activate options ``disallow_incomplete_defs``, ``check_untyped_defs`` and ``strict`` in ``mypy``. (:issue:`1028`)
+- Activate options ``disallow_incomplete_defs``, ``check_untyped_defs``, ``warn_unused_configs``,
+  ``disallow_any_generics``, ``disallow_subclassing_any``, ``disallow_untyped_calls``, ``disallow_untyped_defs``,
+  ``disallow_incomplete_defs``, ``check_untyped_defs``, ``disallow_untyped_decorators``, ``warn_redundant_casts``,
+  ``warn_unused_ignores``, ``warn_return_any``, ``no_implicit_reexport``, ``strict_equality``, ``extra_checks``
+  in ``mypy``. (:issue:`1028`, :issue:`1029`)
```

Comment:
I'm not sure if the set of Mypy configuration options is relevant for people who read the changelog. Something like "stricter Mypy configuration" might be sufficient.